dagger/config_finder/config_processor.py

```diff
@@ -52,8 +50,8 @@ def process_pipeline_configs(self):
             config_path = join(pipeline_config.directory, pipeline_config.config)
 
             _logger.info("Processing config: %s", config_path)
-            config_dict = self._load_yaml(config_path)
-            config_dict = self.overwrite_params(config_dict)
+            config_dict = EnvYAML(config_path).export()
+            config_dict = self.localize_params(config_dict)
```

Comment:
Just a small thing, now we have a `_load_yaml` function that is not used anywhere and we have this two lines replicated which is reading the config with EnvYAML and localising the params. I would put these two lines in the `_load_yaml` function, so we don't have duplication. Wdyt?